vortex-array/src/arrays/struct_/mod.rs

```diff
@@ -104,8 +104,8 @@ impl StructArray {
         for field in fields.iter() {
             if field.len() != length {
                 vortex_bail!(
-                    "Expected all struct fields to have length {length}, found {}",
-                    field.len()
+                    "Expected all struct fields to have length {length}, found {:?}",
+                    fields.iter().map(|f| f.len()).collect_vec(),
```

Comment:
```suggestion
                    "Expected all struct fields to have length {length}, found {}",
                    fields.iter().map(|f| f.len()).format(","),
```
You will also need to import itertools::Itertools